.github/workflows/bootstrap.yml

```diff
@@ -9,7 +9,7 @@ on:
     - cron: '16 2 * * *'
 
 concurrency:
-  group: bootstrap-${{ github.workflow }}-${{ github.event.pull_request.number || github.run_number }}
+  group: bootstrap-${{github.ref}}-${{github.event.pull_request.number || github.run_number}}
```

Comment:
It would be good to document both that *and* the prefixes. They’re only necessary like this because of using reusable workflows forcing GitHub.workflow to the same value for all of them.